Assets/Scripts/Controllers/KeyboardManager.cs

```diff
@@ -22,12 +23,15 @@ public KeyboardManager()
     {
         instance = this;
         mapping = new Dictionary<string, KeyboadMappedInput>();
+        ModalInputFields = new List<InputField>();
 
         TimeManager.Instance.EveryFrameNotModal += (time) => Update();
 
         ReadXmlOrJsonAfterWeDecide();
     }
 
+    public List<InputField> ModalInputFields { get; set; }
```

Comment:
Stylecop complains: `SA1204 : CSharp.Ordering : All static public properties must be placed before all non-static public properties. (Assembly-CSharp)`